entity/FactEntities.xml

```diff
@@ -86,4 +87,73 @@ under the License.
             <key-map field-name="itemCancelledDateDimId" related="dateValue"/>
         </relationship>
     </entity>
+
+    <entity entity-name="OrderAdjustmentFact" package="co.hotwax.bi.fact" group="analytical">
+        <field name="orderAdjustmentId" type="id" is-pk="true"/>
+        <field name="orderId" type="id"/>
+        <field name="orderItemSeqId" type="id"/>
+
+        <!-- Adjustment fields containing amount value -->
+        <field name="shippingCharges" type="currency-amount"/>
+        <field name="shippingSalesTax" type="currency-amount"/>
+        <field name="extShipAdjustment" type="currency-amount"/>
+        <field name="extPriceOverride" type="currency-amount"/>
+        <field name="extFeesAdjustment" type="currency-amount"/>
+        <field name="extRewards" type="currency-amount"/>
+        <field name="extTransAdjustment" type="currency-amount"/>
+        <!-- <field name="extPromoAdjustment" type="currency-amount"/> -->
+        <!-- <field name="salesTax" type="currency-amount"/> -->
+
+        <relationship type="one-nofk" related="co.hotwax.bi.fact.OrderItemFulfillmentFact">
+            <key-map field-name="orderId"/>
+            <key-map field-name="orderItemSeqId"/>
+        </relationship>
+
+        <index name="IDX_OAF_ITEM">
+            <index-field name="orderId"/>
+            <index-field name="orderItemSeqId"/>
+        </index>
+    </entity>
+    
+    <entity entity-name="ReturnItemFact" package="co.hotwax.bi.fact">
+        <field name="returnItemFactId" type="id" is-pk="true"/>
+        <field name="returnId" type="id"/>
+        <field name="returnItemSeqId" type="id"/>
+        <field name="externalReturnId" type="id"/>
+        <field name="dataSourceId" type="text-short"/>
+        <field name="returnDate" type="date-time"/>
+        <field name="returnHeaderTypeId" type="id"/>
+        <field name="destinationFacilityId" type="id"/>
+        <field name="returnChannelEnumId" type="id"/>
+        <field name="returnPrice" type="currency-amount"/>
+        <field name="itemStatusId" type="id"/>
+        <field name="returnDiscountAmt" type="currency-amount"/>
+        <field name="totalTaxRefundAmt" type="currency-amount"/>
```

Comment:
Need to add the following fields:
- return header status
- created by user login
- employee id
- return total
- refunded total amount (sum of all refund transactions on return)
- return item desc
- return item type id (ex Refund)
- return item reason id (ex Did Not Want Item)
- return item RMA qty
- return item received qty
- item received date time
- return shipment id
- received by user login